src/objectModel.ts

```diff
@@ -1,8 +1,4 @@
 // AUTO-GENERATED FILE, DO NOT MODIFY.
-/**
```

Comment:
@mavimaec Could you revert this change please? I think, I added it on the phase of the npm package preparation but didn't update the code generation part.

Reply:
Will do!